synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc11.sh

```diff
@@ -36,13 +36,9 @@ mvn clean install -B -q -DskipTests=true
 export NAME={{ metadata['repo']['distribution_name'].split(':')|last }}
 export VERSION=$(grep ${NAME}: versions.txt | cut -d: -f3)
 
-# V3 generates docfx yml from javadoc
-# generate yml
-mvn clean site -B -q -P docFX
-
-# copy README to docfx-yml dir and rename index.md
-cp README.md target/docfx-yml/index.md
-# copy CHANGELOG to docfx-yml dir and rename history.md
+# cloud RAD generation
+mvn clean javadoc:aggregate -B -q -P docFX
```

Comment:
(Not related to this commit) Line 34 has

```
# compile all packages
mvn clean install -B -q -DskipTests=true
```

The output is cleaned up at line 40. Do we really need line 34?